src/icons/index.ts

```diff
@@ -0,0 +1,8 @@
+// Note: this file has to be manually updated if we add more types of icons.
+// We do not generate it via create-index
```

Comment:
The build process previously was:

1. Target all the SVGs in their respective directory (e.g. `src/svgs/icons`), build them as React components, and output the result to `src/icons`, via `@svgr/cli`
1. Build one large `// @create-index` file in `src/icons` from those files.

This process is not theme-able, though, without stepping in and creating an intermediary file between steps 1 and 2. That also raises the possibility of some messy workflows. So the steps are now driven by `@svgr/webpack` (in step 1) instead of `@svgr/cli`: 

1. Import the SVGs from their respective directory (e.g, `src/icons/icons/svgs`) in TypeScript files that export a themed icon (e.g. `src/icons/icons/arrow.ts`)
1. Build a `// @create-index` file in that respective directory (`src/icons/icons`).
1. Import all exports from that directory (and the others) here, in `src/icons`.